pkg/compiler/function_call_test.go

```diff
@@ -269,3 +274,21 @@ func TestVariadicMethod(t *testing.T) {
 	}`
 	eval(t, src, big.NewInt(42))
 }
+
+func TestJumpOptimize(t *testing.T) {
+	src := `package foo
+	func get1() int { return 1 }
+	func get2() int { get1(); get1(); get1(); get1(); return get1() }
+	func get3() int { return get2() }
```

Comment:
Export them to get more methods in the manifest?

Reply:
Debug info is emitted before generating manifest. But we lose nothing anyway, fixed.